wordle-web/README.md

```diff
@@ -1,3 +1,11 @@
+#Wordle Site Link
+https://brave-forest-08291851e.3.azurestaticapps.net 
+
+#EC Story Ideas
```

Comment:
Extra credit was specifically for ideas related to the wordle game, but that's okay!